src/sentry/static/sentry/app/views/settings/organizationRelays/relays.tsx

```diff
@@ -34,154 +38,237 @@ type State = AsyncComponent['state'] & {
+        onSubmitSuccess={response => {
+          this.successfullySaved(response, t('Successfully added relay public key'));
+        }}
+      />
+    ));
+  }
 
   renderBody() {
     const {relays} = this.state;
+
     return (
       <React.Fragment>
-        <SettingsPageHeader title={t('Relays')} />
-        <Panel>
-          <PanelHeader>{t('Relays')}</PanelHeader>
-          <PanelAlert type="info">
-            {tct('For more details, see [linkToDocs].', {
-              linkToDocs: (
-                <ExternalLink href={RELAY_DOCS_LINK}>
-                  {t('full Relay documentation')}
-                </ExternalLink>
-              ),
-            })}
-          </PanelAlert>
-          <PanelBody>
-            {relays.map(
-              ({public_key, name, created, last_seen, first_seen, last_modified}) => (
-                <Content key={public_key}>
-                  <Info>
-                    <InfoItem>
-                      <Name>{name}</Name>
-                    </InfoItem>
-                    <InfoItem>
-                      <PublicKey>{public_key}</PublicKey>
-                    </InfoItem>
-                    <InfoItem>
-                      <Time label={t('Created:')} date={created} />
-                    </InfoItem>
-                    <InfoItem>
-                      <Time label={t('First Seen:')} date={first_seen} />
-                    </InfoItem>
-                    <InfoItem>
-                      <Time label={t('Last Seen:')} date={last_seen} />
-                    </InfoItem>
-                    <InfoItem>
-                      <Time label={t('Last modified:')} date={last_modified} />
-                    </InfoItem>
-                  </Info>
-                  <Button
-                    title={t('Delete Rule')}
-                    label={t('Delete Rule')}
-                    size="small"
-                    icon={<IconDelete />}
-                  />
-                </Content>
-              )
-            )}
-          </PanelBody>
-          <PanelAction>
-            <Button href={RELAY_DOCS_LINK} target="_blank">
-              {t('Read the docs')}
+        <SettingsPageHeader
+          title={t('Relays')}
+          action={
+            <Button
+              priority="primary"
+              size="small"
+              icon={<IconAdd size="xs" isCircled />}
+              onClick={this.handleOpenAddDialog}
+            >
+              {t('New Relay Key')}
             </Button>
-            <Button onClick={this.handleAdd} priority="primary">
-              {t('Add Relay')}
-            </Button>
-          </PanelAction>
-        </Panel>
+          }
+        />
+        <TextBlock>
+          {tct(
+            `Relay is a relay service built by Sentry. You can run this on-premise for your SDKs or server to customize data scrubbing, buffering retries and more. You can generate relay keys for access. For more on how to set this up, read the [link:docs].`,
```

Comment:
```suggestion
            `Relay is a forwarding service built by Sentry. You can run this on-premise for your SDKs or server to customize data scrubbing, buffering retries and more. You can generate relay keys for access. For more on how to set this up, read the [link:docs].`,
```

'Relay is a relay' is awkward would 'proxy' or 'forwarding' work?

Reply:
There was a discussion about it here: https://www.figma.com/file/et5ZVSG4OTjfUkRAC0bCCb/Proxy-Service-Relay?node-id=0%3A1

In final we decided to keep "relay" here, because It is not really a proxy but a relay... It  relays data to Sentry and processes it along the way...